scripts/plotting/make_postfit_vgen.py

```diff
@@ -19,6 +19,8 @@
 parser.add_argument("--ptll-fit", type=str, default=None)
 parser.add_argument("--ptll-yll-fit", type=str, default=None)
 parser.add_argument("--helicity-fit", type=str, default=None)
+parser.add_argument("--slice-helicity", type=int, default=None)
+parser.add_argument("--slice-charge", type=int, default=None)
```

Comment:
These options might have a "choices" argument to make sure the user only enters valid bins (since you only select a single bin I would also propose renaming them as "--helicity-bin" and "-- charge-bin").

Reply:
In general it would be better to use the usual rebin functionalities e.g. this function: https://github.com/WMass/WRemnants/blob/main/utilities/boostHistHelpers.py#L502-L524 
the the argument can be e.g. --axlim 0 2 to select positive charge and work more generic for any histogram. 
But OK, I understand this script is some high level plotting script and it doesn't need to be perfect, so from my side I leave it to Elisabetta if she wants to generalize things.

Reply:
It's going to need a little bit of generalisation for the theory dependent plotting anyway. I will take a look.